### PR TITLE
Install fork from the cli

### PR DESCRIPTION
**Pull request by carakas**
## Type

- Feature

## Pull request description

Adds a way to install Fork CMS from the CLI

You can install Fork CMS by running the command

    forkcms:install:install

In that case you will need to create a config file in `app/config/cli-install.yml` based on `app/config/cli-install.yml.dist`

For example

    config:
        database:
            hostname: db
            username: forkcms
            password: forkcms
            name:  forkcms
        language:
            multiLanguage: true
            languages:
                - en
                - nl
            defaultLanguage: en
        modules:
            install:
                - Localtion
                - FormBuilder

You can set the email and password of the backend user in the yml config, pass it as an option or by answering the questions in the cli

`bin/console forkcms:install:install --email=test@test.com --password=internet`

### CODE REVIEW

File: docs/01. installation/01. installation.md [hunk 1]
```diff
@@ -26,7 +26,36 @@ Upload the unpacked files into the document_root of your webserver. Make sure yo
...
+        modules:
+            install:
+                - Localtion
+                - FormBuilder
+
+You can set the email and password of the backend user in the yml config, pass it as an option or by answering the questions in the cli
+
+`bin/console forkcms:install:install --email=test@test.com --password=internet`
+
+But if you prefer to install Fork CMS in your browser you can use our installation wizard for that by surfing to your domain (this should forward you to the Fork CMS installer).
```
Comment (WouterSioen) [inline docs/01. installation/01. installation.md]:
```suggestion
But if you prefer to install Fork CMS in your browser, you can use our installation wizard for that by surfing to your domain (this should forward you to the Fork CMS installer).
```

File: docs/01. installation/01. installation.md [hunk 2]
```diff
@@ -26,7 +26,36 @@ Upload the unpacked files into the document_root of your webserver. Make sure yo
...
+            languages:
+                - en
+                - nl
+            defaultLanguage: en
+        modules:
+            install:
+                - Localtion
+                - FormBuilder
+
+You can set the email and password of the backend user in the yml config, pass it as an option or by answering the questions in the cli
```
Comment (WouterSioen) [inline docs/01. installation/01. installation.md]:
```suggestion
You can set the email and password of the backend user in the yml config, pass it as an option or answer the questions in the cli
```

Comment (carakas) [reply docs/01. installation/01. installation.md]: fixed in ab3d83816

File: docs/01. installation/01. installation.md (hunk 1 above)
Comment (carakas) [reply docs/01. installation/01. installation.md]: fixed in ab3d83816

**Pull request merged**